lib/list-group/list-group-item.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cx } from 'emotion';
+import {
+  active,
+  listGroupItem,
+  listGroupItemPrimary,
+  listGroupItemSecondary,
+  listGroupItemSuccess,
+  listGroupItemDanger,
+  listGroupItemWarning,
+  listGroupItemInfo,
+  listGroupItemLight,
+  listGroupItemDark,
+  listGroupItemAction,
+} from '@farvisionllc/emotion-bootstrap/dist/bootstrap';
+
+const colors = {
+  primary: listGroupItemPrimary,
+
```

Comment:
why adding spaces between properties?

Reply:
that was some copy and pasting, plus search and replacing. unintentional.